### PR TITLE
Switch to API v2

### LINKED ISSUE #14: Switch to API v2

### PR DESCRIPTION
**Pull request by ao-picterra**
* Change Picterra API server endpoint to v2
* Retrieve raster list using pagination
* Improve logging during operations

Closes #14.

### CODE REVIEW

File: dist/Client.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ class ValidationError extends Error {
 async function checkResponse(response) {
   if (!response.ok) {
     const text = await response.text();
+    console.error('Error checking response', text);
```
Comment (julienr) [inline dist/Client.js:59]: We are already throwing an error, the `console.error` isn't necessary

File: dist/Client.js [hunk 2]
```diff
@@ -88,20 +89,24 @@ class APIClient {
       fetch = require('node-fetch'); // https://www.npmjs.com/package/node-fetch
 
       Headers = fetch.Headers;
+      console.debug('Node environment');
     } else {
       fetch = window.fetch; // https://caniuse.com/#feat=fetch, https://github.com/github/fetch
 
       Headers = window.Headers; // https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
+
```
Comment (julienr) [inline dist/Client.js:97]: I think in general in JS, you shouldn't call `console.info/console.debug` from library code as this isn't using some proper logging (like logging module in python) and the application using the lib has no way to control the verbosity.

So unless this is really necessary, I'd just chop all the `console` calls here.

I'm not sure what's the best practice for logging in JS, but I don't want to add a logging library as a dependency, so for now it seems the easiest would be to not do logs.

**Review by julienr — CHANGES_REQUESTED** [review]

File: dist/Client.js (hunk 1 above)
Comment (ao-picterra) [reply dist/Client.js]: Checked

File: dist/Client.js (hunk 2 above)
Comment (ao-picterra) [reply dist/Client.js]: Checked

**Review by julienr — APPROVED** [review]

**Pull request merged**